test/dynamo/test_autograd_function.py

```diff
@@ -247,3 +247,26 @@ def test_save_for_bwd(self):
         opt_model = torch._dynamo.optimize("eager", nopython=True)(model)
         x = torch.randn(2, 2, dtype=torch.double, requires_grad=True)
         opt_model(x)
+
+    def test_classmethod(self):
+        class Shake(torch.autograd.Function):
+            @classmethod
+            def forward(cls, ctx, foo):
+                return foo + foo
+
+            @classmethod
+            def backward(cls, ctx, grad_output):
+                return grad_output
+
+        def f(x):
+            return Shake.apply(x)
+
+        x = torch.randn(4, 4, 4, 4, requires_grad=True)
```

Comment:
Thanks for figuring it out, it seems ```requires_grad``` is what makes ```Shake.forward``` a bound method rather than a function.